src/NBitcoin.Tests/BitcoinAddressTest.cs

```diff
@@ -5,14 +5,6 @@ namespace NBitcoin.Tests
 {
     public class BitcoinAddressTest
     {
-        public BitcoinAddressTest()
-        {
-            // These flags may get set due to static network initializers
-            // which include the initializers for Stratis.
-            Transaction.TimeStamp = false;
-            Block.BlockSignature = false;
-        }
-
```

Comment:
Note that we could have left this commit for a subsequent PR. Would have been handy for cross-checking the new derived flags against the old flags.

Reply:
Well there are no new flags anymore, we moved away from flags altogether (apart for the IsPos in the consensus class) but we should try to move away from that as well.